osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -171,6 +172,30 @@ protected override bool OnMouseMove(MouseMoveEvent e)
             return base.OnMouseMove(e);
         }
 
+        protected override bool OnKeyDown(KeyDownEvent e)
+        {
+            switch (e.Key)
```

Comment:
Is it intentional that `Key`s are used here rather than listening for `SelectPrevious`/`SelectNext`/`SelectNextGroup`/`SelectPreviousGroup` actions?

Reply:
It's 100% intentional and would be pointless otherwise. The keys you mention are already handled at a *global level* in `VolumeControlReceptor`. The bindings in this PR are local, when the control is focused, to give more control (generally without alt-prefixes).

Reply:
The `VolumeControlReceptor` handles the volume binding specific ones, which are prefixed with `Alt`. While these are not prefixed with alt and are generally used to traverse through a visible collection of objects (i.e. traversing beatmaps in song select).

Reply:
Are you worried about the UX with these bindings I've added? A specific example may help better.

Reply:
> To reiterate, this is intentional. When the volume controls are focused, I want the arrow keys to work regardless of user bindings. Without the alt- prefix.

I see, missed to realize this part, thinking users are constrained to arrow keys when interacting with the volume overlay, while the keybinds will still work just fine.